parsec/backend/postgresql/migrations/0010_add_pki_certificate_table.sql

```diff
@@ -0,0 +1,51 @@
+ -- Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
```

Comment:
This stuff should also be added to `datamodel.sql` (in the current state, the codebase should not pass the `tests/backend/migrations/test_migrations.py::test_migrations` that checks that once the migrations have been applied, the resulting tables schemas in the database correspond to the one obtained by applying `datamodel.sql` on a blank database)